resources/js/components/showhistory.js

```diff
@@ -62,17 +63,31 @@ const ShowHistory = () =>{
         return <div id='loading'>LOADING...</div>
     }
 
+    const returnPagination = () => {
+        if (!spinner){
+            return <Pagination
+            activePage={paginations?.current_page ? paginations?.current_page : 0}
+            itemsCountPerPage={paginations?.per_page ? paginations?.per_page : 0}
+            totalItemsCount={paginations?.total ? paginations?.total : 0}
+            onChange={(pageNumber) => {fetchResult(pageNumber);}}
+            itemClass="page-item"
+            linkClass="page-link"
+            firstPageText="First Page"
+            lastPageText="Last Page"
+        />
+        }
+    }
```

Comment:
Should have better format:

    const returnPagination = () => {
         if (!spinner) {
             return (
                 <Pagination
                     activePage={paginations?.current_page ? paginations?.current_page : 0}
                     itemsCountPerPage={paginations?.per_page ? paginations?.per_page : 0}
                     totalItemsCount={paginations?.total ? paginations?.total : 0}
                     onChange={(pageNumber) => {fetchResult(pageNumber);}}
                     itemClass="page-item"
                     linkClass="page-link"
                     firstPageText="First Page"
                     lastPageText="Last Page"
                 />
             );
         }
     }